src/components/SortSelector/SortSelector.jsx

```diff
@@ -0,0 +1,128 @@
+              <p className="subtitle-1">First Priority</p>
+              <input
+                type="radio"
+                name="SORTBY_PRIORITY"
+                id="newest"
+                value="hightolow"
+                onChange={(e) =>
+                  filterDispatch({
+                    type: "SORTBY_PRIORITY",
+                    payload: e.target.value,
+                  })
+                }
+                checked={sortByPriority === "hightolow"}
+              />
+            </div>
+          </div>
+          <div
+            className={`${styles.sortOption}  ${
+              sortForm ? styles.showOptions : styles.hideOptions
+            }`}
+          >
+            <span className="material-icons">low_priority</span>
+            <div className="flex-row-wrap align--center gap20">
+              <p className="subtitle-1">Last Priority</p>
+              <input
+                type="radio"
+                name="SORTBY_PRIORITY"
+                id="newest"
+                value="lowtohigh"
+                onChange={(e) =>
+                  filterDispatch({
+                    type: "SORTBY_PRIORITY",
+                    payload: e.target.value,
+                  })
+                }
+                checked={sortByPriority === "lowtohigh"}
+              />
+            </div>
+          </div>
+          {/* Sort By Date */}
+          <div
+            className={`${styles.sortOption}  ${
+              sortForm ? styles.showOptions : styles.hideOptions
+            }`}
+          >
+            <span className="material-icons">date_range</span>
+            <div className="flex-row-wrap align--center gap20">
+              <p className="subtitle-1">Newest</p>
+              <input
+                type="radio"
+                name="SORTBY_DATE"
+                id="oldest"
+                value="Newest"
+                onChange={(e) =>
+                  filterDispatch({
+                    type: "SORTBY_DATE",
+                    payload: e.target.value,
+                  })
+                }
+                checked={sortByDate === "Newest"}
+              />
+            </div>
+          </div>
+          <div
+            className={`${styles.sortOption}  ${
+              sortForm ? styles.showOptions : styles.hideOptions
+            }`}
+          >
+            <span className="material-icons">date_range</span>
+            <div className="flex-row-wrap align--center gap20">
+              <p className="subtitle-1">Oldest</p>
+              <input
+                type="radio"
+                name="SORTBY_DATE"
+                id="oldest"
+                value="Oldest"
+                onChange={(e) =>
+                  filterDispatch({
+                    type: "SORTBY_DATE",
+                    payload: e.target.value,
+                  })
+                }
+                checked={sortByDate === "Oldest"}
+              />
```

Comment:
So for the onChange part of both sections you can make it into a single function
```
const dateInputChange=(e)=>{
filterDispatch({
                    type: "SORTBY_DATE",
                    payload: e.target.value,
             })
}
```
Making it easy to understand while going through later

Reply:
Yes, it is a repeatable piece of code. Will separate it out into a function.